cljfmt/test/cljfmt/config_test.clj

```diff
@@ -1,10 +1,32 @@
 (ns cljfmt.config-test
   (:require [cljfmt.config :as config]
-            [clojure.test :refer [deftest is]]))
+            [clojure.test :refer [deftest is testing]]
+            [clojure.java.io :as io])
+  (:import (java.io File)))
 
 (deftest test-convert-legacy-keys
   (is (= {:indents {'foo [[:inner 0]]}}
          (config/convert-legacy-keys {:indents {'foo [[:inner 0]]}})))
   (is (= {:extra-indents {'foo [[:inner 0]]}}
          (config/convert-legacy-keys {:legacy/merge-indents? true
                                       :indents {'foo [[:inner 0]]}}))))
+
+(deftest test-load-config-extra-configs
+  (testing "`:extra-configs` allows you to import `:extra-indents`"
+    (is (= (config/load-config (io/resource "empty-cljfmt.edn"))
+           (-> (config/load-config (io/resource "cljfmt.edn"))
+               (dissoc :extra-configs)
+               (assoc :extra-indents {})))
+        "should only differ by `:extra-indents` (via `:extra-configs`). All other keys are ignored.")
```

Comment:
Lines should be within 80 characters, and output should be plaintext rather than markdown.

Reply:
fixed, think I got them all?